data/hai/hai reveal 4 middle.txt

```diff
@@ -1375,7 +1378,7 @@ mission "Hai Reveal [A13] A New Elder"
 	on offer
 #		log ``
 		conversation
-			`You are granted access to the secure spaceport, potentially because nobody has filed any paperwork to stop you, and you can't help but notice that the strangely-outfitted Centipede from Teeneep's operations on Darkwaste has made its way here as well. You have arrived late in the day and there is a gloom developing as the sun disappears behind the buildings of the capital as you enter into what you have begun to think of as the 'diplomatic wing' of the government complex. Fortunately, though the place appears nearly deserted, you swiftly run into Alondo and Remington who are sitting in the empty cafeteria area with paired glasses of what appears to be scotch whiskey.`
+			`You are granted access to the secure spaceport, and you can't help but notice that the strangely-outfitted Centipede from Teeneep's operations on Darkwaste has made its way here as well. It is late in the day, and there is a gloom developing as Ya Hai disappears behind the buildings of the capital, when you enter the diplomatic wing of the government complex. Fortunately, though the place appears nearly deserted, you swiftly run into Alondo and Remington who are sitting in the empty cafeteria with paired glasses of what appears to be scotch whiskey.`
 			`	When he sees you Alondo raises a hand and gestures for you to sit with them. "<first>! Bloody hell, come join us."`
```

Comment:
I do think this bit of clarity is relevant, since you'd expect to be blocked right now. Otherwise the elegance of these edits got me thinking on a slightly different path about how to word this though so I hope this sounds good. :)
```suggestion
			`You are granted access to the secure spaceport, potentially because nobody has filed any paperwork to stop you, and you can't help but notice that the strangely-outfitted Centipede from Teeneep's operations on Darkwaste has made its way here as well. It is late in the day and the developing gloom as Ya Hai sinks behind the buildings of the capital underscores the mood when you enter the diplomatic wing of the government complex. The place appears nearly deserted.`
			`	Fortunately though, you swiftly discover Alondo and Remington who are sitting in the empty cafeteria with paired glasses of what appears to be scotch whiskey. When he sees you Alondo raises a hand and gestures for you to sit with them. "<first>! Bloody hell, come join us."`
```